### DISCUSSION TITLE
Don’t require Emacs 29.1

### DISCUSSION THREAD

**Discussion by rswgnu** [kickingvegas/casual-agenda#10] [General]
Hi Charles:

if your dependencies are just versions of Org and Transient, then just fail if those are not met.  If you have other dependencies specific to an Emacs version, you should say what those are.  Some Linux distributions still use Emacs 27.1, so it would be useful to allow for backward compatibility.  Also, the compat package allows these older versions to call the newer Emacs functions, so there is no reason to count them out.

I for example would like to try out your package with Emacs 28.2.

**Comment by kickingvegas** [comment]
@rswgnu Thanks for sharing your input. That said, I’ve chosen Emacs version 29.1 as the minimum requirement for the Casual packages as this release includes the Transient package as a “built-in”. I unfortunately do not have the time nor the resources to verify Casual behavior with an Emacs version earlier than 29.x and as such am resistant to lowering it.

A workaround is to checkout the Casual repo of interest and include it in your `load-path`, provided that you have all the other dependencies resolved.

I understand this might not be what you want to hear, but please consider the weight of QA that I would have to support if I relaxed that requirement.
